app/Providers/RouteServiceProvider.php

```diff
@@ -582,6 +583,7 @@ private function mapApiRoutes()
 
                 Route::get('acars', 'AcarsController@live_flights');
                 Route::get('acars/geojson', 'AcarsController@pireps_geojson');
+                Route::get('acars/sources', [AcarsController::class, 'get_supported_acars_sources']);
```

Comment:
I don't think that we would need this at all, as the sources will be known and static (enums), no need to have a route to get the sources via API.

The module developer will know the proper enum to use anyway.